specifyweb/frontend/js_src/lib/components/WorkBench/WbView.tsx

```diff
@@ -252,867 +202,83 @@ export function WbViewReact({
 
   return (
     <>
```

Comment:
no need for `<>` here?
fragment is only needed if you want to return multiple children from your element - in this case you are returning ReadOnlyContext.Provider (a single element, [that contains other inside but that doesn't matter]), so `<>~` is not needed